runelite-client/src/main/java/net/runelite/client/config/ConfigManager.java

```diff
@@ -514,11 +514,15 @@ static Object stringToObject(String str, Class<?> type)
 		{
 			return Instant.parse(str);
 		}
-		if (type == Keybind.class)
+		if (type == Keybind.class || type == ModifierlessKeybind.class)
 		{
 			String[] splitStr = str.split(":");
 			int code = Integer.parseInt(splitStr[0]);
 			int mods = Integer.parseInt(splitStr[1]);
+			if (type == ModifierlessKeybind.class)
+			{
+				return new ModifierlessKeybind(code, mods);
```

Comment:
Again here, storing mods and then using them for modifierLessKeybind do not makes much sense

Reply:
same thing as above